resources/reservations.py

```diff
@@ -33,7 +33,9 @@
 reservation_fields = {
     'checkin_date': fields.String,
     'checkout_date': fields.String,
```

Comment:
maybe we can also consider using a `Date` type for this

Reply:
See above comment.